src/Components/Body/Body.tsx

```diff
@@ -0,0 +1,54 @@
+import Home from "./Home/Home";
+import {Chat} from "./Chat/Chat";
+import Teams from "./Teams/Chat";
+import Activity from "./Activity/Activity";
+import Apps from "./Apps/Apps";
+import Calls from "./Calls/Calls";
+import Files from "./Files/Files";
+import Help from "./Help/Help";
+import More from "./More/More";
+import SprinklrHub from "./SprinklrHub/SprinklrHub";
```

Comment:
Should create index.ts file in every component directory.
Then we can simply import using path './home'.

Should keep directory naming in camelCase.